selfdrive/thermald.py

```diff
@@ -192,10 +192,12 @@ def thermald_thread():
     if health is not None:
       usb_power = health.health.usbPowerMode != log.HealthData.UsbPowerMode.client
 
-    try:
-      network_type = get_network_type()
-    except subprocess.CalledProcessError:
-      pass
+    # get_network_type is an expensive call. update every 10s
```

Comment:
This really shouldn't live in thermald if it's slow. Mods like this are a bad idea.

Reply:
the timing of thermald is not that important. It's also not that slow, we were talking a few percent cpu, but that's just unnecessary when offroad.

Reply:
Oh, it's an offroad thing for power. Makes sense, but I still dislike mods like this.